grails-app/controllers/au/org/ala/specieslist/AdminController.groovy

```diff
@@ -58,11 +58,11 @@ class AdminController {
         try {
             // retrieve qualified SpeciesListItems for performance reason
             def itemsIds = queryService.getFilterSpeciesListItemsIds(params)
-            def lists = queryService.getFilterListResult(params, false, itemsIds)
+            def lists = queryService.getFilterListResult(params, false, itemsIds, request, response)
```

Comment:
I had a feeling, instead of passing request/response into the method , it would be better off calling  'userId = localAuthService.getJwtUserId(request, response)' to retrieve the userid before the call?

Reply:
https://github.com/AtlasOfLivingAustralia/ala-security-project does not support this use case. When it does these changes can be removed because `authService.getUserId()` will work as required without the need to pass anything.